Missing methods arguments in Keycloak 26.1.0

Attempting to compile the code with Keycloak version 26.1.0 results in the following errors:
```
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /home/nithe/Projects/keycloak-protocol-cas/src/main/java/org/keycloak/protocol/cas/endpoints/AbstractValidateEndpoint.java:[263,31] no suitable constructor found for OAuth2Code(java.lang.String,int,<nulltype>,<nulltype>,java.lang.String,<nulltype>,<nulltype>,java.lang.String)
    constructor org.keycloak.protocol.oidc.utils.OAuth2Code.OAuth2Code(java.lang.String,int,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String) is not applicable
      (actual and formal argument lists differ in length)
    constructor org.keycloak.protocol.oidc.utils.OAuth2Code.OAuth2Code(java.util.Map<java.lang.String,java.lang.String>) is not applicable
      (actual and formal argument lists differ in length)
[ERROR] /home/nithe/Projects/keycloak-protocol-cas/src/main/java/org/keycloak/protocol/cas/CASLoginProtocol.java:[25,8] org.keycloak.protocol.cas.CASLoginProtocol is not abstract and does not override abstract method sendError(org.keycloak.sessions.AuthenticationSessionModel,org.keycloak.protocol.LoginProtocol.Error,java.lang.String) in org.keycloak.protocol.LoginProtocol
[ERROR] /home/nithe/Projects/keycloak-protocol-cas/src/main/java/org/keycloak/protocol/cas/CASLoginProtocol.java:[122,5] method does not override or implement a method from a supertype
```

--- a/pom.xml
+++ b/pom.xml
@@ -22,7 +22,7 @@
 
     <groupId>org.keycloak</groupId>
     <artifactId>keycloak-protocol-cas</artifactId>
-    <version>26.0.8</version>
+    <version>26.1.0</version>
     <name>Keycloak CAS Protocol</name>
     <description />
 
@@ -36,7 +36,7 @@
 
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.release>17</maven.compiler.release>
-        <project.build.outputTimestamp>1736844450</project.build.outputTimestamp>
+        <project.build.outputTimestamp>1737017225</project.build.outputTimestamp>
     </properties>
 
     <dependencies>

--- a/src/main/java/org/keycloak/protocol/cas/CASLoginProtocol.java
+++ b/src/main/java/org/keycloak/protocol/cas/CASLoginProtocol.java
@@ -120,7 +120,7 @@ public class CASLoginProtocol implements LoginProtocol {
     }
 
     @Override
-    public Response sendError(AuthenticationSessionModel authSession, Error error) {
+    public Response sendError(AuthenticationSessionModel authSession, Error error, String errorMessage) {
         if (authSession.getClientNotes().containsKey(CASLoginProtocol.GATEWAY_PARAM)) {
             if (error == Error.PASSIVE_INTERACTION_REQUIRED || error == Error.PASSIVE_LOGIN_REQUIRED) {
                 return Response.status(302).location(URI.create(authSession.getRedirectUri())).build();

--- a/src/main/java/org/keycloak/protocol/cas/endpoints/AbstractValidateEndpoint.java
+++ b/src/main/java/org/keycloak/protocol/cas/endpoints/AbstractValidateEndpoint.java
@@ -260,7 +260,7 @@ public abstract class AbstractValidateEndpoint {
     {
         String key = UUID.randomUUID().toString();
         UserSessionModel userSession = clientSession.getUserSession();
-        OAuth2Code codeData = new OAuth2Code(key, Time.currentTime() + userSession.getRealm().getAccessCodeLifespan(), null, null, redirectUriParam, null, null, userSession.getId());
+        OAuth2Code codeData = new OAuth2Code(key, Time.currentTime() + userSession.getRealm().getAccessCodeLifespan(), null, null, redirectUriParam, null, null, null, userSession.getId());
         session.singleUseObjects().put(prefix + key, clientSession.getUserSession().getRealm().getAccessCodeLifespan(), codeData.serializeCode());
         return prefix + key + "." + clientSession.getUserSession().getId() + "." + clientSession.getClient().getId();
     }

